.github/workflows/lint_test_build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/workflows/rafiki/env-setup
-      - run: pnpm --filter backend build:deps
+      - run: pnpm --filter backend build
```

Comment:
Why this?

Reply:
Because I need backend to be compiled in order to use `./dist/tests/tigerbeetle` in `backend/jest.setup.js`

Reply:
What do you think about one of these alternatives?
https://stackoverflow.com/questions/48318230/configure-jest-global-tests-setup-with-ts-file